sci-libs/symengine/symengine-0.1.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE=""
+
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="Fast symbolic manipulation library, written in C++"
+HOMEPAGE="https://github.com/sympy/symengine"
+SRC_URI="https://github.com/sympy/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="python threads openmp"
+
+DEPEND="
+	dev-libs/jemalloc
+	dev-libs/boost
```

Comment:
@marbre It does not require boost-python, just `date_time`, `system` and `filesystem`.
@kiwifb I'll give the alternate python3 version a try, give me a couple of minutes (I suspect as long as I have `python-single-r1` it will only support one version of python at a time anyway?)